gpu_direct_rdma_access.c

```diff
@@ -838,6 +863,30 @@ struct rdma_device *rdma_open_device_server(struct sockaddr *addr)
     return NULL;
 }
 
+int rdma_reset_device(struct rdma_device *device)
+{
+    if (is_server(device)) {
+        fprintf(stderr, "Method \"rdma_reset_device()\" could be executed only by server side!\n");
+        return EOPNOTSUPP;
+    }
+    struct ibv_qp_attr      qp_attr;
```

Comment:
after moving to ERROR (before RESET)
1. post_send Marker WQE (special wr_id???)
2. poll all CQE's... wait for marker
3. reset rdma_device member {
    struct wr_id_reported       app_wr_id[SEND_Q_DEPTH];
    int                 app_wr_id_idx;
    int                 qp_available_wr;
    khash_t(kh_ib_ah)   ah_hash;
}
4. continue to RESET... rst2rrt